.github/workflows/ci.yml

```
@@ -147,3 +147,36 @@ jobs:
         continue-on-error: ${{ matrix.allowed_failure || false }}
         env: ${{ matrix.env }}
         run: bash ./Tools/ci-run.sh
+        
+
+  code_quality:
+    strategy:
+      # Allows for matrix sub-jobs to fail without canceling the rest
+      fail-fast: false
+    runs-on: ubuntu-18.04
+
+    env:
+      BACKEND: cpp
```

Let's use both the C and the C++ backend for the coverage tests to see if that leaves anything uncovered.
```suggestion
      BACKEND: c,cpp
```